### ISSUE TITLE
PHPUnit 6.5.13 breaks `FilterTest::accept()`

### ISSUE THREAD

**Issue by ThomasLandauer** [Codeception/Codeception#5189]
When running a single test like `codecept run acceptance FooCest:tryBar`, I'm getting the following error since upgrading to `phpunit/phpunit 6.5.13`:

```bash
In FilterTest.php line 12:
                                                                                                                                                    
  Compile Error: Declaration of Codeception\PHPUnit\FilterTest::accept() must be compatible with PHPUnit\Runner\Filter\NameFilterIterator::accept(  
  ): bool
```

The reason is this commit at PHPUnit https://github.com/sebastianbergmann/phpunit/commit/b812332afcc594a683f2334293ad61c6a24b67b0 where the return type `bool` was added to `NameFilterIterator::accept()`.

### Details

* codeception/codeception 2.4.5
* codeception/phpunit-wrapper 6.0.11

**Comment by schmunk42** [comment]
I am seeing the same with codeception 2.3.9 - could you fix your constraints for that, eg. requiring `phpunit < 6.5.13`

**Comment by Naktibalda** [comment]
Sorry, 2.3 branch is no longer maintained, it is up to you to add that constraint to your composer.json.
Also, existing versions can't be updated.

**Comment by Naktibalda** [comment]
Fixed in phpunit-wrapper 6.1.0
